### DISCUSSION TITLE
Capstone Webscrapping

### DISCUSSION THREAD

**Discussion by yuliristiana** [teamalgoritma/community#756] [Q&A] [answered]
Ketika saya akan melihat visualisasi di visual code muncul ->

C:\@MYFOLDER  _  YULIR\ALGORITMA\CAPST - 3A\web_scraping-main>activate capst_webscrapp
'activate' is not recognized as an internal or external command,
operable program or batch file.

itu artinya apa yaa?

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @yuliristiana,

Untuk menjalankan code nya dapat dilakukan di Terminal yang Command Prompt ibu, silahkan diganti melalui tombol (+) di terminal nya, kemudian pilih Command Prompt. Seharusnya akan ada tulisan (base), sehingga ibu dapat melanjutkan code nya dengan activate environment ibu.

Semoga jawaban saya membantu!
